client/components/payment-activity/payment-activity-data.tsx

```diff
@@ -56,9 +56,10 @@ const getSearchParams = ( searchTerms: string[] ) => {
 };
 
 const PaymentActivityData: React.FC = () => {
-	const { paymentActivityData, isLoading } = usePaymentActivityData(
-		getDateRange()
-	);
+	const { paymentActivityData, isLoading } = usePaymentActivityData( {
+		...getDateRange(),
+		timezone: moment( new Date() ).format( 'Z' ),
```

Comment:
This will give the offset (like `+07:00`) which will work just fine for now since we are not worrying about day light saving time etc.

*If* in the future we need a named timezone like `America/Los_Angeles`, we can get it using
```
Intl.DateTimeFormat().resolvedOptions().timeZone
```